src/sentry/api/endpoints/project_release_details.py

```diff
@@ -31,14 +38,38 @@ def get(self, request, project, version):
         :pparam string version: the version identifier of the release.
         :auth: required
         """
+        with_health = request.GET.get("health") == "1"
+        summary_stats_period = request.GET.get("summaryStatsPeriod") or "48h"
+        health_stats_period = request.GET.get("healthStatsPeriod") or ("24h" if with_health else "")
```

Comment:
Do we want to have these on a detail page? I mean there is still datepicker in global selection header there.

Reply:
Right now it returns the same data as the overview page and our APIs are at least somewhat consistent here.